packages/keymaps/src/browser/style/index.css

```diff
@@ -92,6 +74,10 @@
     font-size: calc(var(--theia-ui-font-size1) * 0.8);
 }
 
+.td-scope {
+    text-transform: lowercase;
```

Comment:
Curiosity: Why do you put the tooltip in upper case for this scope column, but all other columns remain exactly as displayed ?

Reply:
Fixed ! :)